### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fabidick22**
Add tools = ["tmux-cssh": "tmux-cluster-ssh", "bom": "Bandwidth Monitor"]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,6 +134,7 @@ for transferring data with URLs.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://en.wikipedia.org/wiki/Lsof"><b>lsof</b></a> - displays in its output information about files that are opened by processes.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="http://www.brendangregg.com/flamegraphs.html"><b>FlameGraph</b></a> - stack trace visualizer.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://github.com/zevv/lsofgraph"><b>lsofgraph</b></a> - small utility to convert Unix lsof output to a graph showing FIFO and UNIX interprocess communication.<br>
+&nbsp;&nbsp;:small_orange_diamond: <a href="https://github.com/tgraf/bmon"><b>bmon - Bandwidth Monitor</b></a> - bmon is a monitoring and debugging tool to capture networking related statistics and prepare them visually in a human friendly way.<br>
```
Comment (trimstray) [inline README.md]: ` - Bandwidth Monitor` is unnecessary and deviating from the convention, the further description explains exactly everything

**Comment by trimstray** [comment]
Thanks a lot!

**Comment by Ramzyr8** [comment]
Accepted

**Review by Bianca-Vision — COMMENTED** [review]
Resolved
EDIT: accidental response to this issue as I had several similar issues through out my repositories today! My apologies!

**Comment by lilmonero** [comment]
Hey hey I need 3 stacks fast. I'm dart can u link me up I'm amazing at subconscious scripting

**Pull request merged**